src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Great info here @AgrimPrasad. I think if polyfills are required in the IE case, we focus on FF first and then tackle IE separately. We could do something in the future like dynamically running polyfills for IE or create a separate compatibility repo that will detect the browser and bridge any API differences. Do you happen to know off the top of your head if there are efforts going into a lib like that?